programs/bench.h

```diff
@@ -16,9 +16,28 @@
 #define ZSTD_STATIC_LINKING_ONLY   /* ZSTD_compressionParameters */
 #include "zstd.h"     /* ZSTD_compressionParameters */
 
+typedef struct {
+    size_t cSize;
+    double cSpeed;   /* bytes / sec */
+    double dSpeed;
+} BMK_result_t;
+
+/* called in cli */
 int BMK_benchFiles(const char** fileNamesTable, unsigned nbFiles, const char* dictFileName,
                    int cLevel, int cLevelLast, const ZSTD_compressionParameters* compressionParams);
 
+/* more options */
+BMK_result_t BMK_benchFilesFull(const char** fileNamesTable, unsigned nbFiles, const char* dictFileName,
```

Comment:
So apparently you already skip intermediate functions when invoking from `paramgrill`,
by calling directly `BMK_benchMem()`.

So I guess the issue is limited to `BMK_benchFilesFull()`.
It simply makes no sense to provide a single "unified" `BMK_result_t ` result to measure multiple compression levels.
I guess it's more appropriate for this function to have a `void` or `int` type.

Also, if there is no user of this function, it might be better to just remove it entirely.
It's a complex function, with a complex behavior, so if it does not serve an immediate identified need, it's just a maintenance tax.